### PR TITLE
Fixed UI typos

### LINKED ISSUE #3088: Minor issues in UI strings
"Integrated address and long payment id can't be used at the same time"

upper case "ID" for consistency:   "... payment ID ..."

-

"Specify ip to bind rpc server"
ip -> IP address
rpc -> RPC

-

"sweep_below <amount_threshold> [mixin] address [payment_id] - Send all unlocked outputs below the threshold to an address"

I think it should be:
"... <amount_threshold> [<mixin>] <address> [<payment_id>] ..."

-

"missing amount threshold"
-->  "... threshold amount ..."

-

"Please try later"
" ... try again ..."

-

"priority must be 0, 1, 2, 3,or 4"
--> "... 3, or ..."

-

"priority must be 0, 1, 2 3,or 4"
--> "... 2, 3, or ..."

- 

"expected at least one payment_id"
--> "... payment ID ..."

-

"The higher the priority, the higher the fee of the transaction."
--> "... transaction fee ..."

-

"locked_transfer [<mixin_count>] <addr> <amount> <lockblocks>(Number of blocks to lock the transaction for, max 1000000) [<payment_id>]"
<addr> --> <address>

-

"invalid language choice passed."
--> "... entered."

-

"Generate new wallet and save it to <arg>"
<arg> --> <file>

-

"Enter new wallet password"
Does it mean "Enter password for new wallet" eller "Enter new password for wallet"

-

"Storing wallet..."
--> "Saving wallet..."

-

"Stored ok"
--> "Saved ok"

-

"Failed to store wallet:"
--> "... save ..."

-

"Use wallet <arg>"
<arg> --> <file>

-

"Failed to initialize wallet rpc server" 
--> "... RPC ..."

- 

"Starting wallet rpc server"
--> "... RPC ..."

-

"Stopped wallet rpc server"
--> "... RPC ..."

-

"Use daemon instance at host <arg> instead of localhost"
<arg> --> <host>

-

"Use daemon instance at port <arg> instead of 18081"
<arg> --> <port>

### PR DESCRIPTION
**Pull request by Maxithi**
Fixes #3088

### CODE REVIEW

**Comment by ordtrogen** [comment]
Man, how awesome is that! :-)

**Comment by moneromooo-monero** [comment]
A lot of those are just arbitrary preferences... "typos", hmm.

**Comment by Maxithi** [comment]
Agree. Even though rpc->RPC, ip->IP, ... may be justified. Once again, made changes based on issue. I can edit/close if you want to.

**Comment by moneromooo-monero** [comment]
I say, change the commit message to say typos and wording tweaks and I'll be OK with it :D

File: src/simplewallet/simplewallet.cpp [hunk 1]
```diff
@@ -158,7 +158,7 @@ namespace
 
   boost::optional<tools::password_container> default_password_prompter(bool verify)
   {
-    return password_prompter(verify ? tr("Enter new wallet password") : tr("Wallet password"), verify);
+    return password_prompter(verify ? tr("Enter new password for the wallet") : tr("Wallet password"), verify);
```
Comment (leonklingele) [inline src/simplewallet/simplewallet.cpp]: `Enter a new password` (`a`)

File: src/wallet/wallet_rpc_server.cpp [hunk 2]
```diff
@@ -2847,9 +2847,9 @@ int main(int argc, char** argv) {
     // if we ^C during potentially length load/refresh, there's no server loop yet
     if (quit)
     {
-      MINFO(tools::wallet_rpc_server::tr("Storing wallet..."));
+      MINFO(tools::wallet_rpc_server::tr("Saving wallet..."));
       wal->store();
-      MINFO(tools::wallet_rpc_server::tr("Stored ok"));
+      MINFO(tools::wallet_rpc_server::tr("Saved ok"));
```
Comment (leonklingele) [inline src/wallet/wallet_rpc_server.cpp]: `Saved ok`? `Successfully saved`!

File: src/wallet/wallet2.cpp [hunk 3]
```diff
@@ -222,7 +222,7 @@ boost::optional<tools::password_container> get_password(const boost::program_opt
 
   THROW_WALLET_EXCEPTION_IF(!password_prompter, tools::error::wallet_internal_error, tools::wallet2::tr("no password specified; use --prompt-for-password to prompt for a password"));
 
-  return password_prompter(verify ? tr("Enter new wallet password") : tr("Wallet password"), verify);
+  return password_prompter(verify ? tr("Enter new password for the wallet") : tr("Wallet password"), verify);
```
Comment (leonklingele) [inline src/wallet/wallet2.cpp]: Same, `Enter a new password` (`a`)

File: src/wallet/wallet_rpc_server.cpp [hunk 4]
```diff
@@ -2877,16 +2877,16 @@ int main(int argc, char** argv) {
...
   }
-  LOG_PRINT_L0(tools::wallet_rpc_server::tr("Stopped wallet rpc server"));
+  LOG_PRINT_L0(tools::wallet_rpc_server::tr("Stopped wallet RPC server"));
   try
   {
-    LOG_PRINT_L0(tools::wallet_rpc_server::tr("Storing wallet..."));
+    LOG_PRINT_L0(tools::wallet_rpc_server::tr("Saving wallet..."));
     wrpc.stop();
-    LOG_PRINT_L0(tools::wallet_rpc_server::tr("Stored ok"));
+    LOG_PRINT_L0(tools::wallet_rpc_server::tr("Saved ok"));
```
Comment (leonklingele) [inline src/wallet/wallet_rpc_server.cpp]: See above, `Saved ok` sounds bad.

**Pull request merged**